markets/retrievaladapter/provider.go

```diff
@@ -38,6 +38,50 @@ func NewRetrievalProviderNode(maddr dtypes.MinerAddress, secb sectorblocks.Secto
 	return &retrievalProviderNode{address.Address(maddr), secb, pp, full}
 }
 
+func (rpn *retrievalProviderNode) GetDealPricingParams(ctx context.Context, storageDeals []abi.DealID) (retrievalmarket.DealPricingParams, error) {
+	resp := retrievalmarket.DealPricingParams{}
```

Comment:
Can we pack as much data as possible as input to the retrieval pricing function? PieceCID, PieceSize, at least.